kernel_gateway/base/handlers.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import notebook.base.handlers as notebook_handlers
+from ..mixins import TokenAuthorizationMixin, CORSMixin
+
+class APIVersionHandler(TokenAuthorizationMixin, 
```

Comment:
Not sure if this API metadata should or should not protected by auth too. For now, erring on the side of safety: protect it.